app/src/main/java/org/dhis2/data/service/SyncPresenterImpl.kt

```diff
@@ -454,7 +458,14 @@ class SyncPresenterImpl(
     }
 
     override fun updateProyectAnalytics() {
-        // TODO: Depends on SDK and Settings Web app
-        analyticsHelper.updateMatomoSecondaryTracker(Any())
+        getSettings()?.let {
+            if (it.matomoID() != null && it.matomoURL() != null) {
```

Comment:
can it be empty String?

Reply:
No, it won't be empty in any case. But now I am wondering how to remove the tracker if the configuration deletes it... I'm going to have a look.

Reply:
oh totally forgot ! good catch